packages/application-shell/src/hooks/use-applications-menu/use-applications-menu.ts

```diff
@@ -30,6 +30,9 @@ export type Config = {
     onError?: QueryFunctionOptions['onError'];
   };
 };
+type TAdditionalEnvironmentProperties = {
+  mcProxyApiUrl?: string;
+};
```

Comment:
We can then simply define a type to extend the app context and include the fields that we expect here to be optionally provided (via the [`additionalEnv`](https://docs.commercetools.com/custom-applications/api-reference/application-config#additionalenv) config field)